src/middlewared/middlewared/plugins/mdns.py

```diff
@@ -53,10 +44,10 @@ def fullname(self):
             self.name.strip('.'),
             self.regtype.strip('.'),
             self.domain.strip('.')
-        ) 
+        )
 
     def to_dict(self):
-        bdict = super(mDNSDiscoveryObject, self).to_dict()
+        bdict = super(mDNSDiscoverObject, self).to_dict()
```

Comment:
Won't `mDNSDiscoverObject.to_dict(self)` be more explicit?

Reply:
How so? `mDNSDiscoverObject` will call itself resulting in an infinite recursion?
Or do you mean `mDNSObject.to_dict(self)`?

Reply:
Sorry, I am wrong, thought it was `mDNSObject` code